src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -377,8 +274,45 @@ export const Tamanos = (): JSX.Element => (
           Opción 3
         </button>
       </div>
-    </div>
+    </Dropdown>
   </div>
 );
 
 Tamanos.story = { name: 'Tamaños' };
+
+export const Options = (): JSX.Element => {
+  const checkboxOptions = [
+    { value: 'checkOne', label: 'Opción 1' },
+    { value: 'checkTwo', label: 'Opción 2' },
+    { value: 'checkThree', label: 'Opción 3' }
+  ];
+
+  const radioOptions = [
+    { value: 'radioOne', label: 'Opción 1' },
+    { value: 'radioTwo', label: 'Opción 2' },
+    { value: 'radioThree', label: 'Opción 3' }
```

Comment:
en Accessibility aparece Ensures every id attribute value is unique. Se podria hacer funcion para que genere arrays con distintos id (idea de Jesus en pasos de tramites :) )
``` 
function createList(size: number, isCheckBox?: boolean): Option[] {
  const list: Option[] = [];
  for (let i = 1; i <= size; i++) {
    const listItem: Option = {
      value: isCheckBox ? `check-${Math.ceil(Math.random() * 1000)}` : `radio-${Math.ceil(Math.random() * 1000)}`,
      label: `Opción ${i}`
    };
    list.push(listItem);
  }
  return list;
}

export const CHECKBOX_CONTENT = createList(3, true);
export const RADIO_CONTENT_1 = createList(3);
export const RADIO_CONTENT_2 = createList(3);
export const RADIO_CONTENT_3 = createList(3);
```

Reply:
listo